app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/modal.tsx

```diff
@@ -18,7 +18,7 @@ export interface ModalProps extends react.PropsWithChildren {
  * background transparency can be enabled with Tailwind's `bg-opacity` classes,
  * like `className="bg-opacity-50"` */
 function Modal(props: ModalProps) {
-    const { children, centered, className } = props
+    const { children, centered = false, className } = props
```

Comment:
Why are we passing `centered` as a prop if we're always setting it to `false`?

Reply:
see above - this is a default value, not an override